tasks/main.yml

```diff
@@ -18,6 +18,7 @@
   register: result
   when: defaultzone.stdout != default_zone
   changed_when: result.stdout == "success"
+  notify: restart firewalld
```

Comment:
If a restart is necessary in case of a default zone change, it would be nice If you could submit this in a separate pull request, thank you.